trikot-viewmodels/viewmodels/src/androidMain/kotlin/com/mirego/trikot/viewmodels/ViewModelBinder.kt

```diff
@@ -35,6 +36,18 @@ fun View.bindViewModel(
             setAlpha(alpha)
         }
 
+        it.isAccessibilityElement.observe(lifecycleOwnerWrapper.lifecycleOwner) { isAccessibilityElement ->
+            importantForAccessibility = if (isAccessibilityElement) View.IMPORTANT_FOR_ACCESSIBILITY_YES else View.IMPORTANT_FOR_ACCESSIBILITY_NO
+        }
```

Comment:
I'm not sure how this will behave with a custom view, that contains sub views. For instance, we don't want a custom button composed of a Frame an icon and a label to have 3 accessibility nodes that can be focused by the screen reader. In a case like this, only the parent Frame view should be important for accessibility, and have a contentDescription that describes the button and its state.

Reply:
If we don't set isAccessibilityElement when initializing the ViewModel, is this code called? If not, it would leave the default value for importantForAccessibility and that would be ok.

Reply:
We could do something like this right? The `MyCustomView` will be accessible but not the subviews.

```
class MyCustomView {
    myLabel: LabelViewModel
    myImage: ImageViewModel

    init() {
        isAccessibilityElement = true
        myLabel.isAccessibilityElement = false
        myImage.isAccessibilityElement = false
    }
}
```

Reply:
> If we don't set isAccessibilityElement when initializing the ViewModel, is this code called?

@dominiclabbe no it's not called since the default value in `MutableViewModel` is 
```
override var isAccessibilityElement = PropertyFactory.never<Boolean>()

override var accessibilityLabel = PropertyFactory.never<String>()

override var accessibilityHint = PropertyFactory.never<ViewModelAccessibilityHint>()
```

Reply:
Ok, I think that could do it then.